compiler/src/dotty/tools/dotc/reporting/messages.scala

```diff
@@ -856,12 +858,13 @@ extends Message(LossyWideningConstantConversionID):
                 |Write `.to$targetType` instead."""
   def explain(using Context) = ""
 
-class PatternMatchExhaustivity(uncoveredCases: Seq[String], tree: untpd.Match)(using Context)
+class PatternMatchExhaustivity(uncoveredCases: Seq[Space], tree: untpd.Match)(using Context)
 extends Message(PatternMatchExhaustivityID) {
   def kind = MessageKind.PatternMatchExhaustivity
 
   private val hasMore = uncoveredCases.lengthCompare(6) > 0
-  val uncovered = uncoveredCases.take(6).mkString(", ")
+  val uncovered = uncoveredCases.take(6).map(SpaceEngine.display(_)).mkString(", ")
+  private val casesWithoutColor = uncoveredCases.map(SpaceEngine.display(_, withoutColors = true))
```

Comment:
wrap this call to display with ctx.withoutColors instead of the withoutColors param